packages/smooth_app/lib/l10n/app_en.arb

```diff
@@ -1,5 +1,9 @@
 {
     "@Utils": {},
+    "sep": "",
+    "@sep": {
+        "description": "Separator just before a colon (':'). Probably only populated in French and empty in other languages."
+    },
```

Comment:
I've read the discussion in the other PR but do we really need this, shouldn't these spaces be added in the Strings itself as you did in the app_fr.arb

Reply:
@M123-dev You're right, we don't need it as long as we have full sentences already localized.

But if we want to build a `String` with very specific business rules, we need this extra label. Off server needed it. And that would be used in the referenced PR.

Should the referenced PR use a colon rather than a comma, that's another story.

Reply:
On the server side, there are many places where we display a field name, a colon and a value. (e.g. "Categories: Coffees"). We do that with something that looks like [field name][sep]: [value], so that in French we can have "Catégories : Cafés".

Reply:
I couldn't say why but that's never happened before in Smoothie.
Perhaps because we rather concatenate `Widget`s than `String`s, or because the most verbose things come directly from the server as KP.